src/fasttext_lite/core.py

```diff
@@ -274,20 +277,20 @@ def __init__(
         self.is_quantized = False
         self.adjusted_labels: Dict[str, str] = {}
 
-    def fit(self, X, Y, labels) -> None:
+    def fit(self, X, y) -> "FastTextMultiOutputClassifier":
```

Comment:
Moving labels to init allows the signature to match across both classifiers and the base class.